packages/manager/tools/component-rollup-config/src/plugins/common/index.ts

```diff
@@ -0,0 +1,15 @@
+const translationNormalize = (text) => {
+  if (text) {
+    const filtered = text
+      .toString()
+      .replace(/&#13;\n/g, ' ') // carriage returns
+      .replace(/&#160;/g, ' ') // spaces
```

Comment:
https://stackoverflow.com/questions/31790440/regex-to-replace-no-break-space < may be more appropriate

Reply:
it's a regex that replace the html code `&#160;` by a whitespace in our json, it's different from replacing a single no-break-space character